app/services/discovery_engine/quality/prometheus_reporter.rb

```diff
@@ -1,7 +1,21 @@
 module DiscoveryEngine::Quality
   class PrometheusReporter
-    def send(_quality_metrics, _label, _label)
-      "to be implemented"
+    def send(quality_metrics, month_label, table_id)
+      metric_collector.record_evaluations(
+        quality_metrics,
+        month_label,
+        table_id,
+      )
+    end
+
+  private
+
+    def metric_collector
+      @metric_collector ||= Metrics::Evaluation.new(registry)
```

Comment:
It looks like the `metric_collector` is being instantiated twice (See [sentry error](https://govuk.sentry.io/issues/6888036985/?alert_rule_id=14659787&alert_type=issue&notification_uuid=72f3b400-39f8-4fc4-8f80-fcf46cfee3a7&project=4505862568935424&referrer=slack))

I don't think we were seeing this error before? So perhaps we are somehow now calling the `metric_collector` more than once?

We probably need to make it a [Singleton class](https://medium.com/@traveling-coder-abhishek/singleton-pattern-explained-from-ruby-on-rails-point-of-view-150e5ee61d47)

We wouldn't have known about the error if you hadn't tested the branch on integration first ⭐

Reply:
I've added two commits to try out a Singleton:

- https://github.com/alphagov/search-api-v2/pull/552/commits/42e9f627fe8f742bf85badb77658d79140060d5e
- https://github.com/alphagov/search-api-v2/pull/552/commits/1770d3a4ab14b8247c36147dcbbabc4fe83aa824

Reply:
That was a nice bit of sleuthing @emmalowe.